<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1440</width>
    <height>960</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" rowspan="17" colspan="4">
     <widget class="QLabel" name="image">
      <property name="maximumSize">
       <size>
        <width>1080</width>
        <height>720</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="4" colspan="3">
     <widget class="QPushButton" name="image_type_menu">
      <property name="text">
       <string>Image Type</string>
      </property>
     </widget>
    </item>
    <item row="1" column="4">
     <widget class="QLabel" name="camera_id_label">
      <property name="text">
       <string>Camera ID:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="5" colspan="2">
     <widget class="QLineEdit" name="camera_id_input">
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="4" colspan="3">
     <widget class="QLabel" name="camera_id_message">
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="3" column="4" colspan="3">
     <widget class="QPushButton" name="color_menu">
      <property name="text">
       <string>Color</string>
      </property>
     </widget>
    </item>
    <item row="4" column="4" colspan="3">
     <widget class="QLabel" name="h_min_label">
      <property name="text">
       <string>H_MIN</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="5" column="4" colspan="3">
     <widget class="QSlider" name="h_min_slider">
      <property name="maximum">
       <number>255</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="6" column="4" colspan="3">
     <widget class="QLabel" name="h_max_label">
      <property name="text">
       <string>H_MAX</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="7" column="4" colspan="3">
     <widget class="QSlider" name="h_max_slider">
      <property name="maximum">
       <number>255</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="8" column="4" colspan="3">
     <widget class="QLabel" name="s_min_label">
      <property name="text">
       <string>S_MIN</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="9" column="4" colspan="3">
     <widget class="QSlider" name="s_min_slider">
      <property name="maximum">
       <number>255</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="10" column="4" colspan="3">
     <widget class="QLabel" name="s_max_label">
      <property name="text">
       <string>S_MAX</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="11" column="4" colspan="3">
     <widget class="QSlider" name="s_max_slider">
      <property name="maximum">
       <number>255</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="12" column="4" colspan="3">
     <widget class="QLabel" name="v_min_label">
      <property name="text">
       <string>V_MIN</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="13" column="4" colspan="3">
     <widget class="QSlider" name="v_min_slider">
      <property name="maximum">
       <number>255</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="14" column="4" colspan="3">
     <widget class="QLabel" name="v_max_label">
      <property name="text">
       <string>V_MAX</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="15" column="4" colspan="3">
     <widget class="QSlider" name="v_max_slider">
      <property name="maximum">
       <number>255</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="16" column="4" colspan="3">
     <widget class="QCheckBox" name="use_morphing_operations">
      <property name="text">
       <string>Use Morphing Operations</string>
      </property>
     </widget>
    </item>
    <item row="17" column="4" colspan="2">
     <widget class="QLabel" name="erode_rect_size_label">
      <property name="text">
       <string>Erode Rectangle Size</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="17" column="6">
     <widget class="QLineEdit" name="erode_rect_size_input">
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="18" column="0" colspan="2">
     <widget class="QCheckBox" name="show_maximum_tag_distance">
      <property name="text">
       <string>Show Maximum Tag Distance</string>
      </property>
     </widget>
    </item>
    <item row="18" column="2" colspan="2">
     <widget class="QCheckBox" name="show_minimum_object_area">
      <property name="text">
       <string>Show Minimum Object Area</string>
      </property>
     </widget>
    </item>
    <item row="18" column="4" colspan="2">
     <widget class="QLabel" name="dilate_rect_size_label">
      <property name="text">
       <string>Dilate Rectangle Size</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="18" column="6">
     <widget class="QLineEdit" name="dilate_rect_size_input">
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="19" column="0">
     <widget class="QLabel" name="maximum_tag_distance_label">
      <property name="text">
       <string>Maximum Tag Distance</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="19" column="1">
     <widget class="QLineEdit" name="maximum_tag_distance_input">
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="19" column="2">
     <widget class="QLabel" name="minimum_object_area_label">
      <property name="text">
       <string>Minmum Object Area:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="19" column="3">
     <widget class="QLineEdit" name="minimum_object_area_input">
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1440</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
